bin/sozo/src/commands/clean.rs

```diff
@@ -9,22 +9,16 @@ use scarb::core::Config;
 #[derive(Debug, Args)]
 pub struct CleanArgs {
     #[arg(short, long)]
-    #[arg(help = "Remove manifests and abis only.")]
-    #[arg(long_help = "Remove manifests and abis only.")]
-    pub manifests_abis: bool,
-
-    #[arg(short, long)]
-    #[arg(help = "Remove artifacts only.")]
-    #[arg(long_help = "Remove artifacts only.")]
-    pub artifacts: bool,
+    #[arg(help = "Removes all the generated files, including scarb artifacts and ALL the \
+                  manifests files.")]
+    #[arg(long_help = "Removes all the generated files, including scarb artifacts and ALL the \
+                       manifests files.")]
```

Comment:
```suggestion
    #[arg(help = "Removes all the generated files, including scarb artifacts and ALL the \
                  manifests files.")]
    #[arg(long_help = "Removes all the generated files, including scarb artifacts and ALL the \
                       manifests files.")]
```

the long help msg is similar to the short one, maybe remove?

Reply:
Fixed in b1f0dcd.